examples/using-graphql-typegen/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "using-graphql-typegen",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Example on how to use GraphQL Typegen in Gatsby",
+  "author": "LekoArts",
+  "scripts": {
+    "develop": "gatsby develop",
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint . --ignore-path .gitignore --ext .ts,.tsx",
+    "lint:fix": "npm run lint --fix"
```

this line should be
```suggestion
    "lint:fix": "npm run lint -- --fix"
```
otherwise `fix` flag will not be passed